demisto_sdk/commands/content_graph/neo4j_service.py

```diff
@@ -6,16 +6,14 @@
 import docker
 import requests
 
-from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
+from demisto_sdk.commands.common.constants import NEO4J_DIR
 from demisto_sdk.commands.common.docker_helper import init_global_docker_client
 from demisto_sdk.commands.common.logger import logger
 from demisto_sdk.commands.content_graph.common import (
     NEO4J_DATABASE_HTTP,
-    NEO4J_FOLDER,
     NEO4J_PASSWORD,
 )
 
-REPO_PATH = CONTENT_PATH.absolute()
 NEO4J_VERSION = "5.13.0"
```

Comment:
i know it is not related but let's move this to constants and have an env library for the version to override it